conf/cassandra.yaml

```diff
@@ -1667,6 +1667,8 @@ drop_compact_storage_enabled: false
 # The two thresholds default to -1 to disable.
 # items_per_collection_warn_threshold: -1
 # items_per_collection_fail_threshold: -1
+# Guardrail to allow/disallow querying with ALLOW FILTERING. Defaults to true.
+# allow_filtering_enabled: true
```

Comment:
On commit I would put this at the end of the guardrails section, instead of between the ones for collections and the one for UDTs. These are related because both of them deal with multi-cell data types.